### PR TITLE
removed stub, updated command to include namespace

### PR DESCRIPTION
**Pull request by markjschreiber**
*Description of changes:*

-  Updates regarding containers needing to support AWS CLI
-  updated NextFlow hello example
-  removed stub, updated command to include namespace

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/orchestration/nextflow/nextflow-overview.md [hunk 1]
```diff
@@ -278,7 +278,8 @@ The `process` definitions in Nextflow scripts should include a couple key parts
 * the `container` directive
 * `cpus` and `memory` directives to define resource that will be used by Batch Jobs
 
-An example defintion for a simple "Hello World" process is shown below:
+!!! note: The container image used to run a process needs to be capable of running the AWS CLI. It **doesn't** have to contain the CLI but it does need shared libraries such as `libz.so.1` which may not be present in very minimal images such as those based on Alpine.
```
Comment (wleepang) [inline docs/orchestration/nextflow/nextflow-overview.md]: Needs to be compatible with `glibc`. Libraries like `libz.so.1` need not be present in the container - just readable from where ever they are mounted.

**Review by wleepang — COMMENTED** [review]
Minor nit on verbiage on libraries Alpine doesn't support. It's mostly that Alpine doesn't normally ship with `glibc` compatibility as an optimization strategy (uses the much smaller `musl` c dist instead).

File: docs/orchestration/nextflow/nextflow-overview.md (hunk 1 above)
Comment (markjschreiber) [reply docs/orchestration/nextflow/nextflow-overview.md]: Clarified in commit below

**Review by wleepang — APPROVED** [review]
lgtm

**Pull request merged**